Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.h

```diff
@@ -49,6 +53,8 @@ class ThreadedCompositor : public CoordinatedGraphicsSceneClient, public ThreadS
 public:
     class Client {
     public:
+        virtual const WebCore::Settings& settings() = 0;
+
```

Comment:
I'm not sure about the settings to be honest, do we really need them, how are we going to expose them? But if we really need settings, I don't think we should be passing the Settings object this way. We can add an enum with flags with the settings we need and pass that instead.

Reply:
Done, I went with an enum that has three posible values.